karma.conf.js

```diff
@@ -81,7 +81,7 @@ module.exports = function (config) {
         client: {
             captureConsole: true,
             mocha: {
-                timeout: process.env.timeout || 2000
+                timeout: process.env.timeout || 6000
```

Comment:
ENV timout is 10000 by the way

Reply:
strange, 3 tests or so were failing for me until i changed this, but i guess that wasn't the cause.

Reply:
you meant ENV timeout is 10000 on our CI, right?
Because on my machine, it uses the second value, and 3 tests keep failing.
So, i'd like to keep this at 6000, so that tests don't fail on my PC.